<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorDialog</class>
 <widget class="QDialog" name="ColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Palette Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="paletteSelectionGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Palette Selection</string>
   </property>
   <widget class="PaletteListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="palettePreviewGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>301</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Palette Preview</string>
   </property>
   <widget class="QLabel" name="bg0PaletteLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BG:</string>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bg0ColorWidget0" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bg0ColorWidget1" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bg0ColorWidget2" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bg0ColorWidget3" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="bp0PaletteLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BP0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="bp1PaletteLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BP1:</string>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bp0ColorWidget0" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bp0ColorWidget1" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bp0ColorWidget2" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bp0ColorWidget3" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bp1ColorWidget0" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bp1ColorWidget1" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bp1ColorWidget2" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="bp1ColorWidget3" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaletteListView</class>
   <extends>QListView</extends>
   <header>src/ColorDialog/PaletteListView.h</header>
  </customwidget>
  <customwidget>
   <class>ColorSelectorWidget</class>
   <extends>QWidget</extends>
   <header>src/ColorDialog/ColorSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
